src/textual/constants.py

```diff
@@ -25,10 +28,18 @@ def get_environ_bool(name: str) -> bool:
     Returns:
         `True` if the env var is "1", otherwise `False`.
     """
-    has_environ = os.environ.get(name) == "1"
+    has_environ = get_environ(name) == "1"
     return has_environ
 
 
+def get_port_for_devtools() -> int:
```

Comment:
This doesn't need to be a callable, as it is never going to change.

Can you create a constant, in the same way as the other symbols in this module.

Reply:
I thought a callable made sense because we had to check if we'd use the default, or not.
Let me try again.